insights/client/client.py

```diff
@@ -243,8 +241,7 @@ def get_branch_info():
     # or we are analyzing a running container/image
     # or tar file, mountpoint, simply return the default branch info
     if (config['offline'] or
-            config['analyze_container'] or
```

Comment:
I still see some references to `analyze_container.` Does that affect execution?

Reply:
NVM, it looks like you merged this config/flag with some other container config/flag stuff correct?

Reply:
Yup. Most of the things from the deprecated `container_mode` switch were changed to `analyze_container` or a related flag. I switched or deleted the remaining ones.